registration/templates/registration/terms.html

```diff
@@ -1,925 +0,0 @@
-{% extends "base.html" %}
```

Comment:
Please check if this page is not being used before removing it.

Reply:
Since, they are not been used, I have removed them :)